src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -87,5 +87,15 @@ public abstract class AnalyzeOptionsBase : CommonOptionsBase
             "optimize",
             HelpText = "Omit redundant properties, producing a smaller but non-human-readable log.")]
         public bool Optimize { get; set; }
+
+        [Option(
+            "trace",
+            Separator = ';',
+            Default = new string[] { },
+            HelpText = "Execution traces, expressed as a semicolon-delimited list, that " +
+                       "should be emitted to the console and log file (if appropriate). " +
+                       "Valid values: ScanTime.")]
+
+        public virtual IEnumerable<string> Traces { get; set; }
```

Comment:
>string [](start = 35, length = 6)

In order to provide a common argument, this must be of 'string' type. This will allow driver apps to replace DefaultTraces with a more expansive enum.